invenio/base/templates/_formhelpers_base.html

```diff
@@ -144,3 +146,17 @@
     {% endif %}
   {% endfor %}
 {% endblock %}{% endmacro %}
+
+{% macro render_radio_field(field) -%}
+<div class="form-group">
+  <label class="control-label" for="name">{{ field.label }}</label>
+  {% for value, label, checked in field.iter_choices() %}
+  <div class="radio">
+    <label>
+      <input type="radio" name="{{ field.id }}" id="{{ field.id }}" value="{{ value }}" {{ "checked" if checked }}>{{ label }}
+      <small class="text-muted">{{ field.description[value] }}</small>
```

Comment:
I think it's better that we create a new widget which is capable of rendering a description text. See e.g. https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/deposit/field_widgets.py#L598